sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -203,7 +216,12 @@ internal static Request ConvertToRequestDocument(Activity activity)
                 }
                 else if (tag.Key == SemanticConventions.AttributeHttpResponseStatusCode)
                 {
-                    httpResponseStatusCode = tag.Value.ToString()!;
+                    requestDocument.ResponseCode = tag.Value.ToString()!;
+                }
+                else if (!ActivityTagsProcessor.s_semanticsSet.Contains(tag.Key) && propertiesCount < MaxPropertiesCount)
```

Comment:
Why do we need this check - !ActivityTagsProcessor.s_semanticsSet.Contains(tag.Key)? It's an additional overhead, we are providing property value based on best effort basis.

Reply:
Because this is a Request telemetry, there will be extra attributes present that don't add any value to the Properties.
For example, all the individual components used to build the full url.

This check is equivalent to only reading the `ActivityTagsProcessor.UnMappedTags`. I don't want to use `ActivityTagsProcessor` here to avoid the extra overhead. As `s_semanticsSet` is a `HashSet`, the overhead is minimal.

Reply:
I would better keep it simple, if there is an ask we could consider it.

Reply:
removed